src/quo2/components/avatars/channel_avatar/style.cljs

```diff
@@ -0,0 +1,26 @@
+(ns quo2.components.avatars.channel-avatar.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def lock-icon-size 12)
+
+(defn outer-container
+  [{:keys [big? color]}]
+  {:width            (if big? 32 24)
+   :height           (if big? 32 24)
+   :border-radius    (if big? 32 24)
+   :justify-content  :center
+   :align-items      :center
+   :background-color (colors/theme-alpha color 0.1 0.1)})
+
+(defn lock-container
+  [{:keys [big?]}]
```

Comment:
It's only one variable, Why do we need a map?

Reply:
I don't use this criteria to decide when to use a map with functions that are components. I see the map as props, like in React, one argument, one map. But I honestly don't follow this rule by the book, nor do I think others should.